application/helpers/mpdf_helper.php

```
@@ -76,8 +76,8 @@ function pdf_create(
     }
 
     // Check if the archive folder is available
-    if ( ! is_dir(UPLOADS_ARCHIVE_FOLDER) || is_link(UPLOADS_ARCHIVE_FOLDER) && ( ! mkdir(UPLOADS_ARCHIVE_FOLDER, '0777') && ! is_dir(UPLOADS_ARCHIVE_FOLDER))) {
-        throw new \RuntimeException(sprintf('Directory "%s" was not created', UPLOADS_ARCHIVE_FOLDER));
+    if ( ! is_dir(uploads_archive_path()) || is_link(uploads_archive_path()) && ( ! mkdir(uploads_archive_path(), '0777') && ! is_dir(uploads_archive_path()))) {
+        throw new \RuntimeException(sprintf('Directory "%s" was not created', uploads_archive_path()));
```

PHPMD (2.15.0)

Missing class import via use statement (line '80', column '19'). (undefined)

(MissingImport)